examples/zxc.rs

```diff
@@ -7,13 +7,19 @@ use bellman::groth16::{
 use bellman::Circuit;
 use bls12_381::{Bls12, Scalar};
 */
+#[cfg(all(feature = "smt", feature = "zok"))]
 use circ::front::zsharp::{self, ZSharpFE};
+
+#[allow(unused_imports)]
```

Comment:
Are the unused_imports and dead_code allows necessary here? Can't they be fixed similarly to the way they were in circ.rs? I could be missing something.

Reply:
Hm, alternatively: can't examples be configured to depend on features in Cargo.toml? If so, given that this executable and zxi totally break in the absence of the smt and zok features, why not just use those flags to gate the whole example in Cargo.toml, rather than using flags to build a stub binary?

Reply:
Added the changes to Cargo.toml! I didn't realize that was capable with rust feature-gating at first. very cool!